client/src/pages/ItemsList.js

```diff
@@ -1,240 +1,213 @@
-import {
-    CssBaseline,
-    TableBody,
-    TableCell,
-    TableHead,
-    TableRow,
-} from '@material-ui/core';
+import MaUTable from '@material-ui/core/Table';
+import { CssBaseline, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
 
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-    padding: 0 40px 40px 40px;
+  padding: 0 40px 40px 40px;
 
-    @media screen and (max-width: 420px) {
-        padding-left: 0.5em;
-        padding-right: 0.5em;
-    }
+  @media screen and (max-width: 420px) {
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+  }
 `;
 
 const Table = ({ columns, data }) => {
-    const {
-        getTableProps,
-        headerGroups,
-        rows,
-        prepareRow
-    } = useTable({
-      columns,
-      data
-    });
-
-    return (
-        <MaUTable {...getTableProps()}>
-            <TableHead>
-                {headerGroups.map(headerGroup => (
-                    <TableRow {...headerGroup.getHeaderGroupProps()}>
-                        {headerGroup.headers.map(column => (
-                            <TableCell {...column.getHeaderProps()}>
-                                {column.render('Header')}
-                            </TableCell>
-                        ))}
-                    </TableRow>
-                ))}
-            </TableHead>
-            <TableBody>
-                {rows.map((row, i) => {
-                    prepareRow(row)
-                    return (
-                        <TableRow
-                            data-row-item-id={row.values._id}
-                            {...row.getRowProps()}
-                        >
-                            {row.cells.map(cell => {
-                                return (
-                                    <TableCell {...cell.getCellProps()}>
-                                        {cell.render('Cell')}
-                                    </TableCell>
-                                )
-                            })}
-                        </TableRow>
-                    )
-                })}
-            </TableBody>
-        </MaUTable>
-    )
+  const { getTableProps, headerGroups, rows, prepareRow } = useTable({
+    columns,
+    data,
+  });
+
+  return (
+    <MaUTable {...getTableProps()}>
+      <TableHead>
+        {headerGroups.map(headerGroup => (
+          <TableRow {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
+              <TableCell {...column.getHeaderProps()}>{column.render('Header')}</TableCell>
+            ))}
+          </TableRow>
+        ))}
+      </TableHead>
+      <TableBody>
+        {rows.map((row, i) => {
+          prepareRow(row);
+          return (
+            <TableRow data-row-item-id={row.values._id} {...row.getRowProps()}>
+              {row.cells.map(cell => {
+                return <TableCell {...cell.getCellProps()}>{cell.render('Cell')}</TableCell>;
```

Comment:
minor: this feels less readable to me than the original formatting

Reply:
I think this one would need to be wrapped as an expression in the original:
```suggestion
                return (
                    <TableCell {...cell.getCellProps()}>
                        {cell.render('Cell')}
                    </TableCell>
                );
```

Reply:
I totally agree, reverted